src/client/Microsoft.Identity.Client/Internal/Constants.cs

```diff
@@ -73,5 +73,11 @@ public static string FormatAdfsWebFingerUrl(string host, string resource)
         {
             return $"https://{host}/.well-known/webfinger?rel={DefaultRealm}&resource={resource}";
         }
+        public enum RequestType
```

Comment:
nit: this needs to be in its own file.

Reply:
@bgavrilMS where would you recommend I put the file?

Reply:
In Http/Retry ?

Reply:
Or just add it in IRetyryPolicyFactory

Reply:
Created PR for this: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5316